pkg/platform/model/buildplanner/buildscript.go

```diff
@@ -52,10 +52,10 @@ func (b *BuildPlanner) GetBuildScript(commitID string) (*buildscript.BuildScript
 	}
 
 	script := buildscript.New()
-	script.SetAtTime(time.Time(resp.Commit.AtTime))
 	if err := script.UnmarshalBuildExpression(resp.Commit.Expression); err != nil {
 		return nil, errs.Wrap(err, "failed to parse build expression")
 	}
+	script.SetAtTime(time.Time(resp.Commit.AtTime))
```

Comment:
This update is to follow the same structure as here: https://github.com/ActiveState/cli/blob/master/pkg/platform/model/buildplanner/build.go#L113

It appears we want to the buildscript `at_time` to be the commit `at_time` even if the buildscript has it's own time set.